packages/frontend/core/src/modules/cloud/services/auth.ts

```diff
@@ -34,6 +35,17 @@ export class AuthService extends Service {
   ) {
     super();
 
+    AIProvider.provide('userInfo', () => {
```

Comment:
not sure if this is the good place to add side effect. Any other suggestions? @EYHN